docs/articles/workflow.md

```diff
@@ -0,0 +1,294 @@
+# Software Provenance Workflow Using OCI Artifact
+
+> :point_right: This article demonstrates an end-to-end workflow for installing a zot registry, pushing and signing an image and artifact, and verifying the results.
```

Comment:
"pushing an image and related artifact (such as an SBOM)"

Drop the ", and verifying..."

Reply:
Done